.github/workflows/bump-all-platforms.yml

```
@@ -3,76 +3,113 @@ name: bump-all-platforms
+        description: "Release kind"
+        required: true
+      pre_release_ident:
+        type: string
+        description: "Pre-release identifier starting at 1 (only used for non-stable releases)"
+        required: false
+        default: "1"
+      update_linux_installer:
+        description: "Update linux installer script"
+        type: boolean
+        required: true
+        default: false
```

actionlint (1.7.12)

[error] input "update_linux_installer" of workflow_call event has the default value "false", but it is also required. if an input is marked as required, its default value will never be used

(events)